specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-05-31/consumption.json

```diff
@@ -0,0 +1,334 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-05-31",
+    "title": "ConsumptionManagementClient",
+    "description": "Consumption management client provides access to consumption resources for Azure Enterprise Subscriptions."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```

Comment:
why we don't have other operations described in this api version such as usageDetails (those are supported in the previous api version)?

Reply:
@anuchandy .. we are not releasing new version for usageDetails and other APIs yet... A new version for those APIs will come in later. 
Simply put the call to new api-version for other APIs will fail as they do not exist. So there is no need to have them in new version of swagger, unless we actually have a new version for those APIs.

Reply:
@anuchandy   We are not releasing new version for usageDetails and other APIs yet... A new version for those APIs will come in later.
Simply put the call to new api-version for other APIs will fail as they do not exist. So there is no need to have them in new version of swagger, unless we actually have a new version for those APIs.

Reply:
@ms-premp - you can add new APis to existing api-version also. I am not sure if i understand spinning up a new api-version for this api unless you are thinking this is in preview (in which case the api-version should be marked preview)